### ISSUE TITLE
Rule: multi-param-spacing

### ISSUE THREAD

**Issue by ilyavolodin** [eslint/eslint#483]
This rule should check function declarations and if, for, while statements for multiple parameters (or Boolean logic) and verify that either all parameters should be on the same line, or they all should be on separate lines:
The following code should produce a warning:

```
if (a && b &&
a === b) {
...
}
```

The following code should not:

```
if (a && b && a===b) {
...
}

if (a &&
b &&
a===b) {
...
}
```

This rule should be off by default.

P.S. This is a suggestion only. A rule like this exists in StyleCop for C#. I find it more annoying then useful most of the times, but I'm sure somebody must like it. Feel free to close if you don't think that it would be useful.
